How-to-Graph-Your-Network.md

```diff
@@ -16,7 +16,32 @@ To manage devices within Cacti, click on the Devices menu item. Clicking Add wil
 
 The field definitions are as follows
 
-||
+###### Table 7-1. Device Field Definitions
+
+Field | Description 
+--- | --- 
+Description | This description will show up in the first column of the device list. You may refer to it e.g. in graph titles
+Hostname | Either an IP address or a hostname. The hostname will be resolved using the standard host resolving mechanisms, e.g. Dynamic Name Services (DNS)
+Host Template | A Host Template is a container for a list of graph templates that will be related to this host. 
+Notes | New with cacti 0.8.7. Add notes to a host to add arbitrary text. 
+Disable Host | Exclude this host from being polled. This is of particular value, if a device is no longer available, but should be kept e.g. as a reference. 
+**Availability/Reachability Options** | 
```

Comment:
This should probably be a new table with a heading like above for Device Field, eg:

```markdown
###### Table 7-2. Availability/Reachability Options

Field | Description 
--- | --- 
```